### DISCUSSION TITLE
Getting rejected on my Schools new campus

### DISCUSSION THREAD

**Discussion by tomichesp** [community/community#46964] [GitHub Education]
My listed school used to have only 2 locations (Madrid and Sevilla). Now they opened a new one in Valencia (where am i). It says im too far from the schools available locations.

If you go to the site, welcome letters or any other documentation it clearly says they have this new branch in Valencia. How to fix it? thanks!

**Comment by jge162** [comment]
You need to submit a [Support ticket](https://support.github.com/request/education?tags=hubberfy_education). We can only discuss here but they can help you. Choose your issue and create ticket. Bests

**Reply by tomichesp** [reply]
thanku, i will!

**Reply by samus-aran** [reply]
Thanks @jge162 👍 

Yes, this type of inquiry is best suited to be directed to support where the internal team can assist with school updates.
